oaknational/Oak-Web-Application
# [CUR-752] Curriculum downloads: filter by child subject & tier

## Description

Music year: 2022

- The curriculum downloads should now create documents that filter based on selected tier or child subject where necessary, and return a complete document with all units when not required


## How to test

1. Go to https://deploy-preview-2652--oak-web-application.netlify.thenational.academy
2. Go to secondary science and filter by child subject and tier
3. Only the relevant info should be available
4. Also can check secondary Maths 
5. Everything else should generate as per previous behaviour

## Checklist
